### PR TITLE
Formatting SD guide (Windows)

### PR DESCRIPTION
**Pull request by ihaveamac**
This adds a guide to format the SD card. First one is for Windows only, ones for Linux and Mac will come after this one receives some polish.

This needs things like warnings (e.g. "This will erase everything!") but I don't know of the best place to put them.

### CODE REVIEW

**Comment by ihaveamac** [comment]
@hacks-guide/3ds

**Comment by ihaveamac** [comment]
Added a warning after the drive letter step.

File: _pages/en_US/formatting-sd-(windows).txt [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+1. If the SD card has any files and folders on it, copy everything to a folder on your computer
+1. Run `guiformat.exe`
+1. Select your SD card's drive letter for "Drive"
+
+	Make sure you choose the correct drive letter, otherwise you might accidentally erase the wrong drive!
+	{: .notice--danger}
+
+1. Select a size for "Allocation unit size"
+  + If the SD card is 32GB or below, choose 32768
+  * If the SD card is above 32GB, choose 65536
```
Comment (jayofdoom) [inline _pages/en_US/formatting-sd-(windows).txt:31]: Folks in discord were saying this threshold was a 128GB SD card -- if it's 32GB, and you're certain of that, let me know and I'll update #1772

**Comment by ihaveamac** [comment]
bump

**Comment by lilyuwuu** [comment]
Looks good, but need to merge mac/linux steps at the same time or else there will be links on the guide that 404

File: _pages/en_US/formatting-sd-(windows).txt (hunk 1 above)
Comment (ihaveamac) [reply _pages/en_US/formatting-sd-(windows).txt:31]: Yes I'm pretty sure it has usually been 64k size for any cards above 32gb, though 64gb with 32k has usually been fine I think.

**Pull request merged**